### PR TITLE
Few small fixes

### LINKED ISSUE #73: Add `sign` endpoint to startup list
https://github.com/vegaprotocol/go-wallet/blob/c1c8da948a74f56c4231286b0a279b24d2bdf5eb/cmd/run_dump_startup.go#L11-L24

### LINKED ISSUE #78: Should `/api/v1/keys:keyid` be `/api/v1/keys/:keyid` ?
Should `/api/v1/keys:keyid` be `/api/v1/keys/:keyid` ?

_Originally posted by @ashleyvega in https://github.com/vegaprotocol/go-wallet/pull/74#discussion_r596902515_

### LINKED ISSUE #79: Improve status REST endpoint
Rather than always returning `{"success": true}`, return:
- `{"success": false}` if zero gRPC nodes can be reached
- `{"success": true}` if at least one gRPC node can be reached

Perhaps use `GetVegaTime` to check if a gRPC node is ok.

### PR DESCRIPTION
**Pull request by jeremyletang**
- Make the healthcheck more useful.
- fix some of the endpoint docs printed at startup.

closes #79 
closes #78 
closes #73

### CODE REVIEW

**Review by ashleyvega — APPROVED** [review]
LGTM

File: wallet/node_forward.go [hunk 1]
```diff
@@ -74,9 +95,16 @@ func (n *nodeForward) Send(ctx context.Context, tx *SignedBundle, ty api.SubmitT
...
 }
 
-func (r *nodeForward) nextClt() api.TradingServiceClient {
-	n := atomic.AddUint64(&r.next, 1)
-	r.log.Info("sending transaction to vega node",
-		zap.String("host", r.nodeCfgs.Hosts[(int(n)-1)%len(r.clts)]))
-	return r.clts[(int(n)-1)%len(r.clts)]
+func (n *nodeForward) nextClt() api.TradingServiceClient {
+	i := atomic.AddUint64(&n.next, 1)
+	n.log.Info("sending transaction to vega node",
```
Comment (candida-d) [inline wallet/node_forward.go:100]: Capital V for Vega

File: wallet/node_forward.go [hunk 2]
```diff
@@ -74,9 +95,16 @@ func (n *nodeForward) Send(ctx context.Context, tx *SignedBundle, ty api.SubmitT
...
+		zap.String("host", n.nodeCfgs.Hosts[(int(i)-1)%len(n.clts)]))
+	return n.clts[(int(i)-1)%len(n.clts)]
+}
+
+func (n *nodeForward) nextCltData() api.TradingDataServiceClient {
+	i := atomic.AddUint64(&n.next, 1)
+	n.log.Info("sending healthcheck to vega node",
```
Comment (candida-d) [inline wallet/node_forward.go:107]: Capital V for Vega (unless it all has to be lowercase)

**Review by candida-d — COMMENTED** [review]
the most minor suggestion possible - capitalisation (non-blocking)

File: wallet/node_forward.go (hunk 1 above)
Comment (jeremyletang) [reply wallet/node_forward.go:100]: well these are logs, I tend to keep them non capitalized

Comment (jeremyletang) [reply wallet/node_forward.go:100]: So if it's not a big deal I would keep them like so.

**Pull request merged**